File: proxy/bottle/py/SystemTest.py
import sys
import bottle
from bottle import route

class SystemTest:
    def __init__(self):
        return

    def listen(self):
        @route('/')
        def index():
            return 'It works!'

        @route('/systemtest')
        def systemtest():
            n = 'Python Version: ' + sys.version + '</br></br>'
            n += 'Bottle Version: ' + bottle.__version__ + '</br></br>'
            return n

File: proxy/node/js/SystemTest.js
module.exports = class SystemTest {
  constructor(app){ this.app = app}
  listen() {
    this.app.get('/', (req, res) => {
      res.send('It Works!')
    })

    this.app.get('/systemtest', (req, res) => {
      let n = 'Node Version: ' + process.version + '</br></br>'
      n += 'V8 Version: ' + process.versions.v8 + '</br></br>'
      res.send(n)
    })
  }
}
